src/frontends/tensorflow/src/op/conv_2d_backprop.cpp

```diff
@@ -14,86 +16,99 @@ namespace tensorflow {
 namespace op {
 
 OutputVector translate_conv_2d_backprop_input_op(const NodeContext& node) {
-    auto ng_filter = node.get_input(1), ng_out_backprop = node.get_input(2);
+    TENSORFLOW_OP_VALIDATION(node, node.get_input_size() >= 2, "Conv2DBackpropInput must have at least three inputs.");
```

Comment:
at least three inputs, but why >= 2?
should be > 2 or >= 3?

Reply:
Great catch! Fixed.